src/chocolatey/GetChocolatey.cs

```diff
@@ -222,6 +222,15 @@ public IEnumerable<T> List<T>()
             var runner = new GenericRunner();
             return runner.list<T>(configuration, _container, isConsole: false, parseArgs: null);
         }
+
+        public int Count()

```

Comment:
Actually, the name of this throws me. It's not clear what this method returns.

This method needs some xml documentation with it as well (looks like I missed calling that out in the list command PR).